src/main/java/uk/ac/ebi/atlas/home/HomeController.java

```diff
@@ -53,4 +59,19 @@ public String getHomePage(Model model) {
 
         return "home";
     }
+
+    private int getCellStats() {
```

Comment:
This method belongs in a DAO class. The controller shouldn’t be concerned with the particular implementation details of how this attribute is obtained.